lesson-10/jquery.js

```diff
@@ -0,0 +1,27 @@
+const express =require("express");
```

Comment:
I found that you like to give a space before equal sign. If you really want to know the standard, it's space on both left and right of the equal sign

Reply:
SONG SUGGESTED NOT TO. :p